run_postgres.sh

```diff
@@ -1,6 +1,14 @@
 #! /bin/bash
 
+# This script should always run as if it were being called from
+# the directory it lives in.
+script_directory=`perl -e 'use File::Basename;
+ use Cwd "abs_path";
+ print dirname(abs_path(@ARGV[0]));' -- "$0"`
+cd $script_directory
+VOLUMES=$(pwd)/volumes_postgres
```

Comment:
Could you add a

```bash
if [ ! -d volumes_postgres ]; then
  mkdir volumes_postgres
fi
```
before this line?

Reply:
Did this fail for you? For me Docker will automatically create the volume if the directory isn't there.

Reply:
Oh ferreal? That's dopeness, I had no idea.